README.md

```diff
@@ -298,6 +298,72 @@ SCANNERS_LIST="grype" BACKEND_HOST=localhost:9999 BACKEND_DISABLE_TLS=true kubec
 ANALYZER_LIST="syft" kubeclarity-cli analyze nginx:latest -o nginx.sbom --merge-sbom inputsbom.xml
 ```
 
+### Remote scanner servers
+
+When running the kubeclarity CLI to scan for vulnerabilties the CLI will need
+to download the relevant vulnerablity DBs to the location where the kubeclarity
+CLI is running. If running the CLI in a CI/CD pipeline this might mean having
```

Comment:
Running the CLI in a CI/CD pipeline will result in downloading the DBs in each run, wasting time and bandwidth.